src/mapeo-manager.js

```diff
@@ -246,7 +246,7 @@ export class MapeoManager {
   }
 
   /**
-   * @returns {Promise<Array<Pick<ProjectValue, 'name'> & { projectId: ProjectPublicId, createdAt?: string, updatedAt?: string }>>}
+   * @returns {Promise<Array<Pick<ProjectValue, 'name'> & { projectId: ProjectPublicId, createdAt?: string, updatedAt?: string, createdBy?: string }>>}
```

Comment:
not sure if `listProjects()` needs to include `createdBy`. how useful is it for a frontend client to have the info?

Reply:
Think for now I would revert this (and the corresponding changes to the tests) and it's it turns out to be useful, do as a follow-up PR

Reply:
reverted on last commit, and now merging